{{cookiecutter.project_name}}/.github/workflows/release.yaml

```diff
@@ -4,6 +4,10 @@ on:
   release:
     types: [published]
 
+defaults:
+  run:
+    shell: bash -e {0} # -e to fail on error
```

Comment:
Is that really necessary? I'd have thought this only affects multi-line commands in `run`.

Reply:
it does. it’ really gross that this isn’t the default, so putting it there will save a lot of people a lot of time.

Reply:
fair enough, let's be safe here.

Reply:
but then shouldn't we go all in with `-euo pipefail` or at least `-eo pipefail`?